chart/tests/ingress_test.yaml

```
@@ -221,3 +221,18 @@ tests:
         nginx.ingress.kubernetes.io/configuration-snippet: |
           more_set_input_headers "X-Forwarded-Host: host.example.com";
           more_set_input_headers "foo: bar";
+- it: should create a http rule with path
+  set:
+    hostname: host.example.com
+  asserts:
+  - equal:
+      path: spec.rules
+      value:
+        - host: host.example.com
+          paths:
+          - service:
+              name: RELEASE-NAME-rancher
+              port:
+                number: 80
+            pathType: ImplementationSpecific
+            path: /
```

test are failing because you are missing `http`
```suggestion
        - host: host.example.com
          http:
            paths:
            - service:
                name: RELEASE-NAME-rancher
                port:
                  number: 80
              pathType: ImplementationSpecific
              path: /
```

---

Thanks, I think I may have forgotten to actually commit or push my last change  🤦 . I certainly remember the test working locally.